LobbyServer2/LobbyServer/LobbyServerProtocol.cs

```diff
@@ -629,6 +632,35 @@ public void HandleGroupLeaveRequest(GroupLeaveRequest request)
             GroupManager.CreateGroup(AccountId);
         }
 
+        public void HandleSelectBannerRequest(SelectBannerRequest request)
+        {
+            PersistedAccountData account = DB.Get().AccountDao.GetAccount(AccountId);
+
+            //  Modify the correct type of banner
+            if(InventoryManager.BannerIsForeground(request.BannerID))
+            {
+                account.AccountComponent.SelectedForegroundBannerID = request.BannerID;
+            }
+            else
+            {
+                account.AccountComponent.SelectedBackgroundBannerID = request.BannerID; 
+            }
+
+            // Update the account
+            DB.Get().AccountDao.UpdateAccount(account);
+
+            // TODO Update friend list and group
```

Comment:
Just call BroadcastRefreshFriendList()

Maybe also call BroadcastRefreshGroup(), but group banners are not implemented yet